lib/generators/flipper/templates/sequel_migration.rb

```diff
@@ -0,0 +1,24 @@
+class CreateFlipperTablesSequel < Sequel::Migration
```

Comment:
How do people know to use this file? Should we hook it up as a rails generator? Or is that bad form since perhaps sequel people may also lean toward sinatra? Does sequel provide any means of using this? Perhaps we just document that people need to require the file? I'd love your thoughts on this.